Earthfile

```
@@ -0,0 +1,247 @@
+    ENV LC_ALL=C.UTF-8
+
+    RUN apt-get update && apt-get upgrade -y && \
+        apt-get install -y build-essential zlib1g-dev \
+        wget unixodbc unixodbc-dev libboost-all-dev cmake g++ \
+        odbc-postgresql postgresql-client ninja-build gnupg apt-transport-https && \
+        apt-get clean
+
+driver:
+    ARG CODE_NAME=focal
+    FROM --build-arg CODE_NAME="$CODE_NAME" +os-base
+
+    # we need an mysql odbc driver for the integration tests
+    # currently the test fail with a newer driver
+    RUN cd /opt && \
+        wget -q https://downloads.mysql.com/archives/get/p/10/file/mysql-connector-odbc-5.1.13-linux-glibc2.5-x86-64bit.tar.gz && \
+        tar xzf mysql-connector-odbc-5.1.13-linux-glibc2.5-x86-64bit.tar.gz && \
+        mysql-connector-odbc-5.1.13-linux-glibc2.5-x86-64bit/bin/myodbc-installer -d -a -n MySQL -t DRIVER=`ls /opt/mysql-*/lib/libmyodbc*w.so` && \
+        rm /opt/*.tar.gz
+
+    # we need an mssql odbc driver for the integration tests
+    RUN wget -q https://packages.microsoft.com/keys/microsoft.asc -O- | apt-key add - && \
+        wget -q https://packages.microsoft.com/config/debian/9/prod.list -O- > /etc/apt/sources.list.d/mssql-release.list && \
+        apt-get update && \
+        ACCEPT_EULA=Y apt-get install msodbcsql17 mssql-tools && \
+        odbcinst -i -d -f /opt/microsoft/msodbcsql17/etc/odbcinst.ini
+
+    # not used for the moment as Exasol is not tested here
+    # RUN cd /opt && \
+    #     wget -q https://www.exasol.com/support/secure/attachment/111075/EXASOL_ODBC-6.2.9.tar.gz && \
+    #     tar xzf EXASOL_ODBC-6.2.9.tar.gz && \
+    #     echo "\n[EXASOL]\nDriver=`ls /opt/EXA*/lib/linux/x86_64/libexaodbc-uo2214lv1.so`\nThreading=2\n" >> /etc/odbcinst.ini && \
+    #     rm /opt/*.tar.gz
+
+python:
+    ARG PYTHON_VERSION=3.8.6
+    ARG CODE_NAME=focal
+    ARG ARROW_VERSION_RULE="<2.0.0"
+    ARG NUMPY_VERSION_RULE=""
+    ARG CONDA_EXTRA=""
+    FROM --build-arg CODE_NAME="$CODE_NAME" +driver
+
+    ENV MINICONDA_URL="https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh"
+    RUN cd /opt && \
+        wget --no-verbose -O miniconda.sh $MINICONDA_URL && chmod +x miniconda.sh
+    ENV MINICONDA=$HOME/miniconda
+    RUN cd /opt && \
+        ./miniconda.sh -b -p $MINICONDA && \
+        . $MINICONDA/etc/profile.d/conda.sh && \
+        conda config --remove channels defaults && \
+        conda config --add channels conda-forge && \
+        conda create -y -q -n turbodbc-dev \
+        gcc_linux-64 \
+        make \
+        ninja \
+        cmake \
+        coveralls \
+        gmock \
+        gtest \
+        gxx_linux-64 \
+        mock \
+        pytest \
+        pytest-cov \
+        python=${PYTHON_VERSION} \
+        unixodbc \
+        boost-cpp \
+        numpy$NUMPY_VERSION_RULE \
+        pyarrow$ARROW_VERSION_RULE \
+        pybind11 \
+        $CONDA_EXTRA \
+        -c conda-forge
```

The channel is already given by the `conda` configuration.
```suggestion
        $CONDA_EXTRA
```

---

Did switch to mambaforge and applied the above suggestions. Thanks, helped a lot.